torch/_inductor/quantized_lowerings.py

```diff
@@ -144,11 +158,25 @@ def int4pack_mm_cpu(
                 (mat1, mat2, group_size, qScaleAndZeros), aten_layout
             ).output_node()
 
+        # define functions to generate example inputs for weight and group size
+        # otherwise, autotuner generates example inputs of all zeros for them
+        def get_example_weight(x: IRNode) -> torch.Tensor:
+            shape = x.get_size()
+            device = x.get_device()
+            return torch.randint(0, 255, shape, dtype=torch.uint8, device=device)
+
+        def get_example_group_size(x: IRNode) -> torch.Tensor:
+            return torch.tensor(qGroupSize, dtype=torch.int64)
+
         return autotune_select_algorithm(
             "_weight_int4pack_mm_for_cpu",
             choices,
             [mat1, mat2, group_size, qScaleAndZeros],
             aten_layout,
+            input_gen_fns={
+                1: get_example_weight,
+                2: get_example_group_size,
```

Comment:
I think the weight and group_size are all constant buffer, can we use
```
            input_gen_fns = {
                1: lambda x: V.graph.constants[x.get_name()],  # packed weight
                2: lambda x: V.graph.constants[x.get_name()],  # group size
            }
```

Reply:
Thanks. Modified.